services/grafana/dashboards/space-rover-rev1.json

```diff
@@ -25,7 +25,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "Prometheus",
-      "description": "",
+      "description": "The battery level percentage of the space rover. If the space rover isn't reporting the percentage, or is a wired model, this will show 0%. ",
```

Comment:
If the rover is wired, then I believe this should show 100%

Reply:
Ha, fair enough. I can change it. My reasoning was if someone saw it was at 100% they might unplug it 😂